### DISCUSSION TITLE
Obsidian note aliases

### DISCUSSION THREAD

**Discussion by hi-learn-triz-and-usit** [srid/emanote#39] [Ideas]
See the Obsidian [docs](https://help.obsidian.md/How+to/Add+aliases+to+note) for motivation and suggested syntax.

**Comment by srid** [comment]
Is Obsidian the only application that supports this? For eg., what happens if I use a notebook with Obsidian-style aliases in VSCode with vscode-memo or related extension? The alias link is considered broken, right?

What's the implication for static site generation (or is there no impact here?) ?

**Reply by hi-learn-triz-and-usit** [reply]
- vscode-memo does not: https://github.com/svsool/vscode-memo/issues/256
- dendron does: https://wiki.dendron.so/notes/d758fe38-558c-4f20-a091-cac146a781cb.html#adding-a-link-alias
- org-roam does: https://www.orgroam.com/manual.html#File-Titles
- roam... doesn't seem to, but I honestly can't tell with their documentation. It seems to be a highly requested feature, though

The implication for static site generation _may_ be about what to generate as the link text; that may end up being a separate issue, though.

**Reply by srid** [reply]
The dendron one you linked to talks about link text (i.e, `[[foo|some text]]`), not aliases.

> You can add an alternative **title** to the link by creating an alias. For example, add `the project|` to your link to change the **name in the preview**

So it looks like only Obsidian and org-roam supports this feature.

**Reply by j-steinbach** [reply]
Wikipedia does this ([Redirects](https://en.wikipedia.org/wiki/Wikipedia:Redirect)), and in Dendron it is on the [list](https://github.com/dendronhq/dendron/issues/92). 

Aliases are very useful for abbreviations (UK -> United Kingdom), of for multi-language topics. For example I write my notes in either english or german and want both to link to the same file (for example a note called `homework` might also have the aliases `Hausaufgaben` and `hausis`.

**Reply by j-steinbach** [reply]
Another useful application of aliases is the realm of science, e.g. bibliography. If I have a note/paper called `Emanote: Spiritual successor to neuron, based on Ema`, then I want to alias it to `srid21emanote`. This allows me to turn any mentions of `srid21emanote` into links, without having to write the whole cumbersome title of the paper.
